neo/framework/Common.cpp

```diff
@@ -2637,6 +2637,13 @@ void idCommonLocal::LoadGameDLLbyName( const char *dll, idStr& s ) {
 			s.AppendPath(dll);
 			gameDLL = sys->DLL_Load(s);
 		}
+
+		// if not found in the bundle's directory, try in the bundle itself
```

Comment:
weird, why does this turn up here, it was already merged in https://github.com/dhewm/dhewm3/commit/ae6997aef5b4203ffee4cdc9db6542987830ac82